src/tests/validity/generic_array_in_argument.ab

```diff
@@ -0,0 +1,10 @@
+// Output
+// 1 2 3
+// Hello Привіт こんいちは
+
+fun test(arr: []): Null {
+        echo arr
+}
+
+test([1, 2, 3])
+test(["Hello", "Привіт", "こんいちは"])
```

Comment:
is it really necessary to use non-ansi characters here? im pretty sure it would break with different platforms

Reply:
This is totally arbitrary, thanks for raising the issue.